components/hab/src/cli.rs

```diff
@@ -184,15 +184,16 @@ struct SubSupRun {
                 possible_values = &["standalone", "leader"])]
     topology: Option<habitat_sup_protocol::types::Topology>,
     /// The update strategy; [default: none] [values: none, at-once, rolling]
-    // TODO (DM): this should use possible_values = &["none", "at-once", "rolling"]
+    // TODO (DM): this should set a default_value and use possible_values = &["none", "at-once",
+    // "rolling"]
```

Comment:
I _really_ wish our update strategy was just `at-once` or `rolling`, with the `none` being handled by an `Option`. :cry: